sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -147,6 +147,16 @@ directive:
     }
 ```
 
+### Make CORS allow null values
+It should be possible to pass null for CORS to update service properties without changing existing rules.
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions.BlobServiceProperties
+  transform: >
+    $.properties.Cors["x-az-nullable-array"] = true;
```

Comment:
Can we add a note about this new property at the top of https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/swagger/Generator/readme.md

I'm trying to keep track of all the extensions we've added.